OpenRA.Mods.TS/UtilityCommands/LegacyTilesetImporter.cs

```diff
@@ -61,7 +61,7 @@ public void Run(ModData modData, string[] args)
 					var section = file.GetSection("TileSet{0:D4}".F(tilesetGroupIndex));
 
 					var sectionCount = int.Parse(section.GetValue("TilesInSet", "1"));
-					var sectionFilename = section.GetValue("FileName", "");
+					var sectionFilename = section.GetValue("FileName", "").ToLowerInvariant();
```

Comment:
It does not affect the current tilesets because mixes are case-insensitive in all OS. It affects loose files alone.